frontend/selectors/index.js

```diff
@@ -1,20 +1,54 @@
 import { createSelector } from 'reselect';
-import { getCurrentProduct } from '@shopgate/pwa-common-commerce/product/selectors/product';
+import { getProduct } from '@shopgate/pwa-common-commerce/product/selectors/product';
 
+/**
+ * Appends required params for legacy image server.
+ * @param {string} href
+ * @returns {string}
+ */
+const decorateLegacyImageUrl = (href) => {
+  try {
+    const url = new URL(href);
+    url.searchParams.set('w', '880');
+    url.searchParams.set('h', '880');
+
+    return url.toString();
+  } catch (err) {
+    console.error(err);
+    return href;
+  }
+};
+
+/**
+ * Sanitizes image url by detecting legacy urls and appending requided params.
+ * @param {string} url Url.
+ * @returns {string}
+ */
+const sanitizeImageUrl = (url) => {
+  if (!(url && typeof url === 'string' && url.length)) {
+    return '';
+  }
+
+  if (url.startsWith('https://img-cdn.shopgate.com')) {
+    return decorateLegacyImageUrl(url);
+  }
+
+  return url;
+}
 /**
  * Params for share button
  * @return {Array}
  */
 export const getShareParams = createSelector(
-  getCurrentProduct,
+  getProduct,
   (currentProduct) => {
     if (!currentProduct) {
       return null;
     }
 
     const shareParams = {
       title: currentProduct.name,
-      imageURL: currentProduct.featuredImageUrl,
+      imageURL: sanitizeImageUrl(currentProduct.featuredImageUrl),
```

Comment:
we should use this helper here instead. 
https://github.com/shopgate/pwa/blob/2badef2fa5157ff91456931d48bb937334f42fee/libraries/common/helpers/data/index.js#L44

Reply:
Oh, didn't know that "getActualImageSource" does this kind of sanitisation. 

But still, I'd prefer to use my own since the library version won't sanitise if the url contains any "?" char. Which is less reliable in my opinion.